third_party/nvidia/backend/compiler.py

```diff
@@ -218,7 +218,7 @@ def make_llir(src, metadata, options, capability):
         if options.extern_libs:
             paths = [path for (name, path) in options.extern_libs]
             llvm.link_extern_libs(llvm_mod, paths)
-        llvm.optimize_module(llvm_mod, llvm.OPTIMIZE_O3)
+        llvm.optimize_module(llvm_mod, llvm.OPTIMIZE_O3, "")
```

Comment:
We can drop the `""` entirely given we have a default value there?

Reply:
I tried that but python complains about the type
https://github.com/triton-lang/triton/actions/runs/9072828063/job/24928949117
Do you have any clue why it does not work?

Reply:
you probably need `py::arg`? https://pybind11.readthedocs.io/en/stable/basics.html#default-args

Reply:
Thank you for the pointer :)